### DISCUSSION TITLE
'Could not allocate memory' error running testbed.exe

### DISCUSSION THREAD

**Discussion by jessekirbs** [NVlabs/instant-ngp#431] [Q&A]
Hello,

After running `testbed.exe --scene /path/` I get the following error:

```
13:54:27 INFO     Loading NeRF dataset from
13:54:27 INFO       C:\Tools\Volumetric\instant-ngp\data\bedrock\transforms.json
13:54:28 SUCCESS  Loaded 70 images of size 4032x3024 after 0s
13:54:28 INFO       cam_aabb=[min=[-1.02051,-0.939806,0.440189], max=[2.01879,2.53002,0.578044]]
13:54:28 ERROR    Uncaught exception: Could not allocate memory: C:\Tools\Volumetric\instant-ngp\dependencies\tiny-cuda-nn\include\tiny-cuda-nn/gpu_memory.h:112 cudaMalloc(&rawptr, n_bytes+DEBUG_GUARD_SIZE*2) failed with error out of memor
```
I'm on Windows 10, NVIDIA RTX 2060, i9 9900k. Any guidance would be greatly appreciated. Thank you!

**Comment by HumbleShaw** [comment]
The GPU memory is not enough, downsampling the image resolution to 1920x1080 may be helpful.

**Comment by jessekirbs** [comment]
Thank you, @QinlongHuang ! That got it working. Do you have any idea how to determine how much memory the software requires for different datasets?

**Comment by HumbleShaw** [comment]
I'm not sure. Maybe you should have a test on your GPU to find the maximum points you can put in the GPU, which means ( image_nums x image_w x image_h ) rays x N points sampled per ray, i.e., ( N x image_nums x image_w x image_h ) points.
